eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/FeedCommand.java

```diff
@@ -20,6 +19,7 @@ public FeedCommand(NoticeService noticeService) {
     }
 
     @Execute
+    @Permission("eternalcore.feed.self")
```

Comment:
Moim zdaniem nie potrzebna końcówka **.self**.
```suggestion
    @Permission("eternalcore.feed")
```